pallets/parachain-staking/src/inflation.rs

```diff
@@ -99,8 +108,9 @@ pub fn round_issuance_range<T: Config>(round: Range<Perbill>) -> Range<BalanceOf
 	}
 }
 
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
```

Comment:
This is repeated a lot (effectively adding `Serialize` and `Deserialize` to `no_std`). Is it actually needed in no_std now? It could have a big impact on code size...

Reply:
Yes, I removed it because it was necessary.